### ISSUE TITLE
TypeError: feature.getFailureException is not a function

### ISSUE THREAD

**Issue by jameswtc** [webdriverio/wdio-sauce-service#40]
Hello all,

I'm getting this error when running the the test using wdio testrunner:

```
TypeError: feature.getFailureException is not a function
    at SauceService.afterStep (/xxxxxxxxxxxxxxxx/node_modules/wdio-sauce-service/build/sauce-service.js:94:53)
    at execHook (/xxxxxxxxxx/node_modules/wdio-sync/build/index.js:166:35)
```

There is a similar fix on wdio-browserstack-service package, see https://github.com/itszero/wdio-browserstack-service/issues/12

I guess this is the same problem that needs fixing?

Best regards,

James

**Comment by christian-bromann** [comment]
@jameswtc yes, would you create a PR?

**Comment by jameswtc** [comment]
I see that this fix is already done on the master branch but has yet to be released.

**Comment by christian-bromann** [comment]
it was released with `0.4.4`
